Intel_SighPP_CMake/CMakeLists.txt

```diff
@@ -3,18 +3,8 @@
 #
 cmake_minimum_required (VERSION 3.8)
 
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
```

Comment:
Idk much about CMake but should this be removed?

Reply:
I don't know either. I think I'll just keep these two lines.